Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -129,4 +142,229 @@ private function getItemsPerPage(ResourceInterface $resource, Request $request)
 
         return $resource->getItemsPerPageByDefault();
     }
+
+    /**
+     * Determines whether output walkers should be used.
+     *
+     * @param QueryBuilder $queryBuilder
+     *
+     * @return bool
+     */
+    private function useOutputWalkers(QueryBuilder $queryBuilder)
```

Comment:
I just thought it could be useful in Doctrine ORM (of course, that's subject to the opinion of Doctrine maintainers).